lib/encoder.py

```diff
@@ -549,6 +549,10 @@ def AllScoredVariants(self, videofile, vary):
                                                          videofile).encodings)
     return encodings
 
+  def ChangeValue(self, name, new_value):
```

Comment:
I wanted to be consistent with the function on the OptionValueSet class. Filing an issue that this name needs to be changed globally (39 occurences).